en/developer_guide/smartAdmin.rst

```diff
@@ -0,0 +1,81 @@
+SmartAdmin
+==========
+
+`SmartAdmin`_ is an responsive admin theme which use Bootstrap realize by Myorange.
+
+Open Orchestra back office uses the specific design of SmartAdmin
+and only some of this JavaScript components (JarvisWidget and SmartNotification).
+
+SmartConfirm
+============
+
+Description
+-----------
+
+The function ``smartConfirm`` allows to display a confirmation message.
+It uses the function ``SmartMessageBox`` of component SmartNotification.
+It is called during various events like the deleting of an item (nodes, blocks, contents, ...).
+And must be used in replacement of the standard JavaScript confirm.
+
+A confirmation message is composed of a logo, a title, a description and two buttons (Yes/No).
+
+.. image:: ../../images/smartconfirm.png
+
+The function ``smartConfirm`` takes 4 parameters:
+
+1.  the logo css class.
+2.  the message title.
+3.  the message description.
+4.  an array of callbacks functions which are called when buttons are clicked and
+the parameters of callbacks functions.
+
+The array must contain the keys ``yesCallback`` and ``NoCallback``
+which refer respectively to the functions managing the click event on Yes and No buttons.
+And the key ``callBackParams`` is an array which contains the parameters for callbacks functions.
```

Comment:
I have the feeling that this explanation should be part of the point 4 of the previous list. Otherwise i don't see what it refers to.